docker/docker.go

```diff
@@ -64,12 +64,12 @@ func DoRollingUpdateContainer(dClient *client.Client, imageCfg *container.Config
 		logrus.Infof("[%s] Container %s is not running on host [%s]", plane, containerName, hostname)
 		return nil
 	}
-	logrus.Infof("[%s] Pulling Image on host [%s]", plane, hostname)
-	err = PullImage(dClient, hostname, imageCfg.Image)
+	logrus.Infof("[%s] Checking image on host [%s]", plane, hostname)
+	err = UseLocalOrPull(dClient, hostname, imageCfg.Image)
 	if err != nil {
 		return err
 	}
-	logrus.Infof("[%s] Successfully pulled [%s] image on host [%s]", plane, containerName, hostname)
+	logrus.Infof("[%s] Image [%s] exists on host [%s]", plane, containerName, hostname)
```

Comment:
this info message will show that image exist even if it pulled the image, it should reflect what happened exactly (pulled or found)

Reply:
maybe add the message in UseLocalOrPull function, instead of using it for each occurrence

Reply:
The reasoning here is that you would lose `plane` in the log message when you move it to `UseLocalOrPull`. Passing `plane` for just the log doesn't seem the right way, even as printing the message as return from the function.